### ISSUE TITLE
Training on the nuscenes mini dataset - questions

### ISSUE THREAD

**Issue by harsanyidani** [tjiiv-cprg/EPro-PnP#74]
Hi!

I have some questions in connection with training your detection network (epropnp_det_basic.py config unchanged) from strach. 

1. Why is the weight file I got (634.88 MB) much bigger than the provided epropnp_det_basic.pth (212.82 MB)?

2. Below you an find the evaluation results (1)(2). **Are these normal after 12 epochs?** I find the construction vehicle, barrier and trailer results strange. Maybe they are under represented in the dataset and they have less than 10% precision and accuracy in every operating point so the area under the precision-recall curve is considered 0 and the same logic for the TP metrics? I also attached some info about the classes in the mini dataset (3) and the results on the training set after 12 epochs (4)(5).
(1): ![1](https://github.com/tjiiv-cprg/EPro-PnP/assets/99925221/31323038-d9c0-4afc-9b4e-963c8bba0986)
(2): ![2](https://github.com/tjiiv-cprg/EPro-PnP/assets/99925221/ee50ce42-8bd1-4622-a093-8daf1829bb0c)
(3): ![3](https://github.com/tjiiv-cprg/EPro-PnP/assets/99925221/3578e24a-0399-4e4e-83f3-69129e1c138f)
(4): ![4](https://github.com/tjiiv-cprg/EPro-PnP/assets/99925221/d6823114-27a3-495c-a3e7-229a95a75d5c)
(5): ![5](https://github.com/tjiiv-cprg/EPro-PnP/assets/99925221/3537ae82-3425-48b5-9e53-40939423a44b)

3. As I was inspecting the tensorflow logs I noticed there are no evaluation losses computed and tracked? Why is that? Is the measurement of nuscenes metrics enough to detect overfitting? 

I also included the logs and tensorflow logs from my training as a zip file:
[logs.zip](https://github.com/tjiiv-cprg/EPro-PnP/files/11630013/logs.zip)

Thanks in advance for your help!

**Comment by Lakonik** [comment]
Hi! 

1. The checkpoint contains not only weights but also optimizer states, so it should be about 3x the size
2. EPro-PnP-Det does not work very well with small training set, so the results can be pretty bad when trained on the mini set
3. Validation loss calculation is disabled by default (otherwise training would take a lot more time), and I think the metrics are good enough for detecting overfitting

**Comment by harsanyidani** [comment]
Thanks!
